app/src/main/java/com/theQuake/quakeInfo/EarthquakeActivity.java

```diff
@@ -111,6 +113,41 @@ protected void onCreate(Bundle savedInstanceState) {
         // so the list can be populated in the user interface
         earthquakeListView.setAdapter(mAdapter);
 
+        // Getting SwipeContainerLayout
+        swipeLayout = findViewById(R.id.swipe_container);
+
+        // Adding Listener
+
+        swipeLayout.setOnRefreshListener(new SwipeRefreshLayout.OnRefreshListener() {
+            @Override
+            public void onRefresh() {
+                // Get a reference to the LoaderManager, in order to interact with loaders.
+                LoaderManager loaderManager = getLoaderManager();
+
+                // Initialize the loader. Pass in the int ID constant defined above and pass in null for
+                // the bundle. Pass in this activity for the LoaderCallbacks parameter (which is valid
+                // because this activity implements the LoaderCallbacks interface).
+                loaderManager.initLoader(EARTHQUAKE_LOADER_ID, null, EarthquakeActivity.this);
```

Comment:
It says wrong third argument type. It requires `LoaderCallbacks` in `EarthquakeActivity.this`.